README.md

```diff
@@ -1,37 +1,61 @@
 
-Beside merging in platform specific SDKs, the following changes were made compared to original mapbox projects:
+This project originated as a fork of Mapbox GL Native, before their switch to a non-OSS license in December 2020. For more information, see: [`FORK.md`](./FORK.md).
 
-* The code was upgraded so that it can be built using latest clang compiler / Xcode 12.
-* CI/CD was migrated from CircleCI to GitHub Actions. 
-* Along with GitHub releases, binaries are distributed as follows:
-    * The iOS binaries distribution was upgraded from fat packages to Swift package containing XCFramework.
-    * The Android binaries are distributed to GitHub maven package repository.
+**Call for Bounties💰** If you have ideas for new features in MapLibre, you can now nominate them for the MapLibre Bounty Program at https://maplibre.org/news/2022-10-16-call-for-bounties/
 
-> The mapbox-gl-native was forked from [d60fd30 - mgbl 1.6.0](https://github.com/mapbox/mapbox-gl-native/tree/d60fd302b1f6563e7d16952f8855122fdcc85f73), mapbox-gl-native-ios from [a139216](https://github.com/mapbox/mapbox-gl-native-ios/commit/a139216) and mapbox-gl-native-android from [4c12fb2](https://github.com/mapbox/mapbox-gl-native-android/commit/4c12fb2c)
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/649392/211550776-8779041a-7c12-4bed-a7bd-c2ec80af2b29.png" alt="Android device with MapLibre" width="24%">   <img src="https://user-images.githubusercontent.com/649392/211550762-0f42ebc9-05ab-4d89-bd59-c306453ea9af.png" alt="iOS device with MapLibre" width="25%">
+</p>
+
+## Getting Started
+
+To get started with MapLibre GL Native, go to your platform below.
 
-**Call for Bounties💰** If you have ideas for new features in MapLibre, you can now nominate them for the MapLibre Bounty Program at https://maplibre.org/news/2022-10-16-call-for-bounties/
 ## Documentation
-MapLibre GL Native documentation is a work in progress. To know about the current state of MapLibre and it's path forward, 
-read the Markdown book. To know about API references for Android and iOS, please follow the API Reference links.
 
-1. [MapLibre GL Native Markdown Book](https://maplibre.org/maplibre-gl-native/docs/book/)
-2. [MapLibre GL Native iOS API Reference](https://maplibre.org/maplibre-gl-native/ios/api/)
-2. [MapLibre GL Native Android API Reference](https://maplibre.org/maplibre-gl-native/android/api/)
+The documentation of MapLibre GL Native is a work in progress. To get an architectural overview and to learn about the current state of the project and its path forward read the [MapLibre GL Native Markdown Book](https://maplibre.org/maplibre-gl-native/docs/book/). See below for platform-specific documentation.
+
+## Platforms
+
+- [Android](platform/android/README.md)
+- [iOS](platform/ios/platform/ios/README.md)
+
+### Community maintained
```

Comment:
This gives a bit of a strange message. Android and iOS are also community maintained, they just have more financial backing, let's say.

Also I suspect macOS and iOS SDKs could (and should) really be one SDK supporting two platforms (but please correct me if this is not the case here).

Also I'm not 100% sure that linux works as an independent "SDK". In MapLibre Native nomenclature platform is not directly a platform to target.

Reply:
As I see it these are not neccesarily SDKs, just platforms MapLibre works on or has an example project for.

MapLibre GL Native for iOS and macOS is one SDK, but of the two only iOS is a core project (@wipfli can correct me here). 

You are right w.r.t. the wording of community! I updated it. How do you like the current solution?

Reply:
Personally I see no reason why `macOS` should not be a core project as the overlap would be very large. As a developer I do not appreciate separate macOS and iOS SDKs.

Can you remind me where the project tiers are defined? 😊

Reply:
Here: https://github.com/maplibre/maplibre/blob/main/PROJECT_TIERS.md

And see this discussion: https://github.com/maplibre/maplibre/discussions/186

Reply:
Well, it defines the tier but it does not list platforms explicitly. We can discuss tomorrow, but I think macOS and iOS SDKs should be merged.

Reply:
@ntadej Alright, let's clarify this tomorrow. Thanks for your input. I resolved the other things you mentioned now. Let me know if you would like to see anything else changed.

Reply:
We may also want to clarify that there is a core that works on specific platforms and then we have some actual SDKs (but again we can do this in another PR).

Reply:
@ntadej Can this be resolved? :angel: